src/FluentlyHttpClient/Middleware/LoggerHttpMiddleware.cs

```diff
@@ -60,7 +60,12 @@ public async Task<FluentHttpResponse> Invoke(FluentHttpMiddlewareContext context
 			else
 			{
 				var requestContent = await request.Message.Content.ReadAsStringAsync();
-				_logger.LogInformation("Pre-request... {request}\nContent: {requestContent}", request, requestContent);
+				_logger.LogInformation(
+					"Pre-request... {request}\nContent: {requestContent}\nHeaders: {headers}",
```

Comment:
what do you think if we switch headers and content order, so headers are more visible?